.github/workflows/ci.yml

```
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [3.1, '3.0', 2.7, 2.6, 2.5]
+        ruby-version: [3.2, 3.1, '3.0', 2.7]
     steps:
       - uses: actions/checkout@v2
```

Unrelated, but this is also out of date
```suggestion
      - uses: actions/checkout@v3
```

---

Dang it, forgot to push this commit to add this. Follow up PR coming